update-serve-file.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
In case you would write down a issue which describes what your composer issues are I might get something done for you.